src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -121,6 +133,13 @@ public class MicrobeEditor : NodeWithInput, ILoadableGameState, IGodotEarlyNodeR
     /// </summary>
     private bool organelleDataDirty = true;
 
+    /// <summary>
+    ///   Similar to organelleDataDirty but with the exception that this is only set false when the editor
+    ///   membrane mesh has been redone. Used so the membrane doesn't have to be rebuild everytime when
+    ///   switching back and forth between structure and membrane tab (wihout editing organelle placements).
+    /// </summary>
+    private bool membraneOrganellePositionsAreDirty;
```

Comment:
Shouldn't this default to true like organelleDataDirty?
I guess for the future this is useful, but right now membrane generation should be cheap enough to even happen each frame.